### PR TITLE
Adding 'amount' to OrderRequest, improving README.md and testing SSL

### PR DESCRIPTION
**Pull request by ralphavalon**
- Adding 'amount' to OrderRequest (OrderAmountRequest)
- Improving README.md (Showing correctly how to catch Exceptions and correcting indentation)
- Testing SSL (There was a TODO to create a test to SSLSocketFactory with SSLSupport. So, I tested this.)

### CODE REVIEW

File: src/main/java/br/com/moip/request/OrderRequest.java [hunk 1]
```diff
@@ -20,6 +21,16 @@ public OrderRequest ownId(final String ownId) {
 
         return this;
     }
+    
+    public OrderAmountRequest getAmount() {
+		return amount;
```
Comment (joaolucasl) [inline src/main/java/br/com/moip/request/OrderRequest.java]: Wrong indentation

File: src/main/java/br/com/moip/request/OrderRequest.java [hunk 2]
```diff
@@ -20,6 +21,16 @@ public OrderRequest ownId(final String ownId) {
...
+	}
+    
+    public OrderRequest amount(OrderAmountRequest amount) {
```
Comment (joaolucasl) [inline src/main/java/br/com/moip/request/OrderRequest.java]: Indentation, too

File: src/main/java/br/com/moip/request/SubtotalsRequest.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+		return discount;
+	}
+	
+	public SubtotalsRequest discount(Integer discount) {
+		this.discount = discount;
+		
+		return this;
+	}
+	
+	@Override
```
Comment (joaolucasl) [inline src/main/java/br/com/moip/request/SubtotalsRequest.java]: Indentation, too

**Comment by ralphavalon** [comment]
Style issues fixed

**Pull request merged**